processor/src/main/java/com/infinum/jsonapix/processor/extensions/AnnotationSpecExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package com.infinum.jsonapix.processor.extensions
```

Comment:
I extracted this into a seperate file, let me know if it should be somewhere else.

Reply:
I think you can have it in `AnnotationExtensions`

Reply:
Moved it there.